pkl-cli/pkl-cli.gradle.kts

```diff
@@ -33,9 +112,23 @@ val stagedMacAarch64Executable: Configuration by configurations.creating
 val stagedLinuxAmd64Executable: Configuration by configurations.creating
 val stagedLinuxAarch64Executable: Configuration by configurations.creating
 val stagedAlpineLinuxAmd64Executable: Configuration by configurations.creating
+val modulepath: Configuration by configurations.creating
+val compileClasspath: Configuration by configurations.getting {
+  extendsFrom(modulepath)
+}
+
+private fun stagedDir(dir: String): File = layout.buildDirectory.dir(dir).get().asFile
 
 dependencies {
-  compileOnly(libs.svm)
+  compileOnly(libs.kotlinStdlib)
+
+  // JPMS module path
+  modulepath(libs.kotlinStdlib)
+  modulepath(libs.jlineReader)
+  modulepath(libs.svmTruffle)
+  modulepath(libs.truffleApi)
+  modulepath(libs.truffleRuntime)
+  modulepath(projects.pklCore)
```

Comment:
Use the `modulepath` configuration in `pkl-cli` to move dependencies to the `--module-path`.